custom/conf/app.example.ini

```diff
@@ -927,14 +927,18 @@ ROUTER = console
 ;USE_COMPAT_SSH_URI = false
 ;;
 ;; Close issues as long as a commit on any branch marks it as fixed
-;; Comma separated list of globally disabled repo units. Allowed values: repo.issues, repo.ext_issues, repo.pulls, repo.wiki, repo.ext_wiki, repo.projects
+;; Comma separated list of globally disabled repo units. Allowed values: repo.issues, repo.ext_issues, repo.pulls, repo.wiki, repo.ext_wiki, repo.projects, repo.packages
 ;DISABLED_REPO_UNITS =
 ;;
-;; Comma separated list of default repo units. Allowed values: repo.code, repo.releases, repo.issues, repo.pulls, repo.wiki, repo.projects.
+;; Comma separated list of default new repo units. Allowed values: repo.code, repo.releases, repo.issues, repo.pulls, repo.wiki, repo.projects, repo.packages.
 ;; Note: Code and Releases can currently not be deactivated. If you specify default repo units you should still list them for future compatibility.
 ;; External wiki and issue tracker can't be enabled by default as it requires additional settings.
 ;; Disabled repo units will not be added to new repositories regardless if it is in the default list.
-;DEFAULT_REPO_UNITS = repo.code,repo.releases,repo.issues,repo.pulls,repo.wiki,repo.projects
+;DEFAULT_REPO_UNITS = repo.code,repo.releases,repo.issues,repo.pulls,repo.wiki,repo.projects,repo.packages
+;;
+;; Comma separated list of default forked repo units.
+;; The set of allowed values and rules are the same as DEFAULT_REPO_UNITS.
+;DEFAULT_FORK_REPO_UNITS = repo.code,repo.pulls
```

Comment:
So this is a break change from before?

Reply:
And looks like a repository cannot reenable releases, there is no option in repository setting.

Reply:
In terms of breaking changes, it depends what is considered breaking. Someone could have a script that uses the API to fork a repo and then create an issue in that repo.  Or some company might have a documented forking workflow where everyone is used to getting certain units enabled in the fork that are now disabled by default. Those things would break.

The only way to avoid that is making this opt-in in the configuration. Which is trivial to do, we could make `DEFAULT_FORK_REPO_UNITS` equal to `DEFAULT_REPO_UNITS` if it's not specified.  But then you arguably have a suboptimal default configuration.

Reply:
I think making `DEFAULT_FORK_REPO_UNITS` equal to `DEFAULT_REPO_UNITS` is a good idea.

Reply:
To clarify, do you want `DEFAULT_FORK_REPO_UNITS` equal to `DEFAULT_REPO_UNITS` only when `DEFAULT_REPO_UNITS` is specified in the configuration, or also when `DEFAULT_REPO_UNITS` was not specified?

The latter is what would completely preserve compatibility.

Reply:
For re-enabling releases, I made #22671.

Reply:
If #22671 merged, I prefer to add a `break` label in this PR and go ahead.